tests/test_models.py

```diff
@@ -522,12 +522,14 @@ def test_enroll_failure_with_no_order(self, enrollment_order_mock, enrollment_ap
             "Client Error", content=json.dumps({"message": "Enrollment error"}).encode()
         )
         enrollment_api_client_mock.return_value.get_course_enrollment.return_value = None
+        enterprise_enrollment_count = EnterpriseCourseEnrollment.objects.count()
         with LogCapture(level=logging.ERROR) as log_capture:
             enterprise_customer_user.enroll('course-v1:edX+DemoX+Demo_Course', 'verified')
             enrollment_api_client_mock.return_value.enroll_user_in_course.assert_called_once()
             analytics_mock.track.assert_not_called()
             enrollment_order_mock.assert_not_called()
             assert 'Enrollment error' in log_capture.records[0].getMessage()
+            self.assertEqual(EnterpriseCourseEnrollment.objects.count(), enterprise_enrollment_count)
```

Comment:
Can we add a comment or better variable name so later we can know that why did we add this check?